docroot/sites/facilities.uiowa.edu/modules/facilities_core/src/Plugin/Block/AccessibilityLinks.php

```diff
@@ -60,6 +62,9 @@ public function build() {
       $url = 'https://uiadmin.maps.arcgis.com/apps/webappviewer/index.html?id=' . $link['id'] . '&query=Buildings,BuildingNumber,' . $building_number;
       $list_markup .= '<a class="bttn bttn--transparent bttn--tertiary bttn--small" href="' . $url . '">' . $link['label'] . '<i class="fa-solid fa-' . $link['icon'] . ' "></i></a>';
     }
+    if ($energy_dashboard != NULL) {
```

Comment:
```suggestion
    if (!empty($energy_dashboard)) {
```
If we don't have a value set, the `->getString()` should return an empty string, and `empty()` and `== NULL` evaluate the same way in this case, but I think the `empty()` makes it more clear when reading that an empty string is expected and not a true null value.

Reply:
Ahh that makes sense, thank you for the explanation! I've updated the code to use `empty()`.